webapp/src/components/toggleSwitch/index.tsx

```diff
@@ -15,10 +14,10 @@ const ToggleSwitch = ({
     active,
     onChange,
     label,
-    labelPositioning = ToggleSwitchLabelPositioning.Left,
+    labelPositioning = 'left',
 }: ToggleSwitchProps): JSX.Element => (
-    <div className={`toggle-switch-container d-flex align-items-center ${labelPositioning === ToggleSwitchLabelPositioning.Right && 'flex-row-reverse'}`}>
-        {label && <span className={labelPositioning === ToggleSwitchLabelPositioning.Left ? 'toggle-switch-label--left' : 'toggle-switch-label--right'}>{label}</span>}
+    <div className={`toggle-switch-container d-flex align-items-center ${labelPositioning === 'right' && 'flex-row-reverse justify-content-end'}`}>
+        {label && <span className={labelPositioning === 'left' ? 'toggle-switch-label--left' : 'toggle-switch-label--right'}>{label}</span>}
```

Comment:
Shouldn't we be using constants here for adding default positioning?

Reply:
we can use utilize TS type for this, using constant will lead to always importing that constant while passing as prop whereas type doesn't

Reply:
I guess while passing the props it'll be okay to use a string literal. But here in the component, we should use constant because this is something that's not expected to change.

Reply:
same is the case with type if a type is fixed then it's not going to change, we must utilise TS

Reply:
`type` can be `left | right`, but by default, we want `left` here.